### ISSUE TITLE
Maccy nao desce nem sobe, não consigo acessar as configurações porque o único modo parece que é em baixo do app. Já desinstalei e instalei novamente, mas o erro persiste.

### ISSUE THREAD

**Issue by Ricardoat** [p0deje/Maccy#892]
### Before Submitting Your Bug Report

- [ ] I have verified that there isn't already an issue reporting the same bug to prevent duplication.
- [ ] I have seen the [FAQ](https://github.com/p0deje/Maccy?tab=readme-ov-file#faq).

### Maccy Version (see 'About' window)

2.0.0

### macOS Version

14.6.1 

### Maccy Settings

```Shell
Maccy nao desce nem sobe, não consigo acessar as configurações porque o único modo parece que é em baixo do app. Já desinstalei e instalei novamente, mas o erro persiste.
```

### Description

Maccy nao desce nem sobe, não consigo acessar as configurações porque o único modo parece que é em baixo do app. Já desinstalei e instalei novamente, mas o erro persiste.

### Steps to Reproduce

star + alt + c
scroll doesnt work

**Comment by p0deje** [comment]
Please record a screen video showing the problem.

**Comment by Ricardoat** [comment]
https://github.com/user-attachments/assets/fc84df91-cf82-44d8-872c-7d2604babaeb

I'm recording in a vertical screen because it's the only way I can expand Maccy enough to access the entire list. On horizontal screens, I can't do that. I also can't access the settings from the icon—it seems the only way to do that is from the bottom, but since the screen is too small, I can't reach the settings. The cursor and scroll aren't working either, which prevents me from scrolling up or down to find copies. This issue started after the update, and I was really satisfied before, but now I can't use it because I can't scroll. If you notice, when I use the vertical screen and manage to apply the whole thing, only about a third of the list can be scrolled. The other two-thirds remain static. The markers have also changed position; the whole layout is different now.

**Comment by Ricardoat** [comment]
I'm also using Instantview to be able to use an extra monitor in Mac M1, but even when I turn off the Instantview app, the issue persists. This makes me think the bug is really from the update.

**Comment by p0deje** [comment]
@Ricardoat This is definitely related to the update, before Maccy 2.0 it was not possible to resize the window. Your list looks very unusual because the pinned items are sorted with regular items. Or are these all pinned items?

**Comment by Ricardoat** [comment]
Yes. I don't know what happened, the layout changed, the biggest problem is that I can't scroll up and down, this makes it impossible for me to use the application.

**Comment by p0deje** [comment]
Do you have all these items pinned?

**Comment by Ricardoat** [comment]
yes

**Comment by Ricardoat** [comment]
i can send you another video if u want

**Comment by p0deje** [comment]
I haven't found a way to make the list scrollable without breaking other pieces. Since nobody else reported a similar problem, I can suggest you avoid having so many pinned items. Is there any particular reason and use-case for having them?

**Comment by Romachamp** [comment]
I can resize window as well, have similar issue.

**Comment by Ricardoat** [comment]
> I haven't found a way to make the list scrollable without breaking other pieces. Since nobody else reported a similar problem, I can suggest you avoid having so many pinned items. Is there any particular reason and use-case for having them?

I believe that using prompts will become common in the future, so having the ability to retrieve those prompts in the most intuitive and quick way possible is very useful. In the previous version, everything was fine because I could scroll through both the pinned and variable lists, not just the variable one, as it is in version 2.0. It's unfortunate

**Comment by euwei7** [comment]
I also encountered this problem with the latest version

**Comment by hichemfantar** [comment]
i believe the same applies to the preview #1075
